app/controllers/account/search_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Account::SearchController < Account::AccountController
+  def index
+    render json: {
+      workspaces: current_user.workspaces.search_workspaces(params[:search]),
```

Comment:
what you return in JSON? the whole objects?
why?
return fields only which you need.
I don't think you need anything else except "id" and "title" correct?